integration-box/yahoo-dmp/README.md

```diff
@@ -1,26 +0,0 @@
-# Workflow definition to integrate Yahoo DMP
```

Comment:
seems readme is removed while code is renewed. don't we need to have new readme?